app/web_ui/src/routes/(app)/docs/rag_configs/[project_id]/table_rag_config_row.svelte

```diff
@@ -0,0 +1,160 @@
+<script lang="ts">
+  import type { RagConfigWithSubConfigs } from "$lib/types"
+  import { formatDate } from "$lib/utils/formatters"
+  import RunRagControl from "./run_rag_control.svelte"
+  import {
+    embedding_model_name,
+    model_name,
+    provider_name_from_id,
+  } from "$lib/stores"
+  import {
+    ragProgressStore,
+    type RagConfigurationStatus,
+  } from "$lib/stores/rag_progress_store"
+
+  export let rag_config: RagConfigWithSubConfigs
+  export let project_id: string
+  $: rag_progress = $ragProgressStore.progress[rag_config.id || ""]
+
+  let row_hovered = false
+
+  // Calculate percentages for progress bar
+  $: total_docs = rag_progress?.total_document_count || 0
+  $: completed_pct =
+    total_docs > 0
+      ? Math.round(
+          ((rag_progress?.total_document_completed_count || 0) / total_docs) *
+            100,
+        )
+      : 0
+
+  function status_to_badge_props(status: RagConfigurationStatus) {
+    switch (status) {
+      case "complete": {
+        return {
+          text: "Complete",
+          color: "badge-primary",
```

Comment:
primary color is reserved for clickable buttons, which these aren't. 

See comment below which I think resolves this (combining with the view details button)